pcsx2/IopBios.cpp

```diff
@@ -657,8 +672,8 @@ namespace R3000A
 			if (is_host(full_path))
 			{
 				const std::string path = full_path.substr(full_path.find(':') + 1);
-				ghc::filesystem::path folder_path{host_path(path)};
-				bool suceeds = ghc::filesystem::remove(folder_path);
+				const ghc::filesystem::path folder_path{host_path(path)};
+				const bool suceeds = ghc::filesystem::remove(folder_path);
```

Comment:
typo, succeeds would be correct, but succeeded might be a better way to word it?

Reply:
Applied!